External/Plugins/CodeRefactor/Controls/MoveDialog.cs

```diff
@@ -174,5 +176,10 @@ protected override void OnKeyDown(KeyEventArgs e)
                     break;
             }
         }
+
+        private void OnTreeSelectedIndexChanged(object sender, EventArgs e)
+        {
+            this.processButton.Enabled = true;
```

Comment:
I'm not sure this really adds too much value, because it would be interesting to disable it when no item is selected, but when the list becomes empty and no item is selected the even doesn't trigger. It would be nicer to fully make it enabled or disabled on every case.

PS: I didn't even try this dialog ever before nor knew where it was displayed it heh.